presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/TransactionCachingJdbcClient.java

```diff
@@ -31,9 +32,9 @@
 {
     private final Cache<TableStatisticsCacheKey, TableStatistics> statisticsCache;
 
-    public TransactionCachingJdbcClient(JdbcClient delegate, Duration cachingTtl)
+    public TransactionCachingJdbcClient(JdbcClient delegate, Set<SessionPropertiesProvider> sessionPropertiesProviders, Duration cachingTtl)
     {
-        super(delegate, cachingTtl, true);
+        super(delegate, sessionPropertiesProviders, cachingTtl, true);
```

Comment:
i believe JDBC connectors do not support transactions, so "transaction" is actually per query cache.
As such, there is no way to change session properties _during_ a transaction, so you can avoid overheader and pass empty set here (with appropriate explanatory comment in the code)